src/data/set/basic.lean

```diff
@@ -1733,6 +1742,10 @@ theorem prod_mono {s₁ s₂ : set α} {t₁ t₂ : set β} (hs : s₁ ⊆ s₂)
   s₁.prod t₁ ⊆ s₂.prod t₂ :=
 assume x ⟨h₁, h₂⟩, ⟨hs h₁, ht h₂⟩
 
+lemma forall_prod_set {p : α × β → Prop} :
+  (∀ x ∈ s.prod t, p x) ↔ ∀ (x ∈ s) (y ∈ t), p (x, y) :=
```

Comment:
This is defeq to the following lemma `prod_subset_iff` right? Maybe use that lemma as the proof of this one?